warehouse/templates/accounts/two-factor.html

```diff
@@ -89,7 +89,17 @@ <h2>{% trans %}Authenticate with an app{% endtrans %}</h2>
               <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
               {% endif %}
             </label>
-            {{ totp_form.totp_value(autocapitalize="off", autocomplete="off", inputmode="numeric", pattern="[0-9]*", required="required", spellcheck="false", class_="form-group__field", **{"aria-describedby":"totp-errors"}) }}
+            {{ totp_form.totp_value(
+              autocapitalize="off",
+              autocomplete="off",
+              inputmode="numeric",
+              pattern="[0-9]*",
+              required="required",
+              spellcheck="false",
+              class_="form-group__field",
+              aria_describedby="totp-errors",
+              autofocus=not has_webauthn,
```

Comment:
https://wtforms.readthedocs.io/en/2.3.x/widgets/#wtforms.widgets.html_params specifies that boolean params are handled correctly (Fals = not rendered, True = rendered as `=""`